From aa1d8ce5f8bdcca83e8c04044caa47d2add9e065 Mon Sep 17 00:00:00 2001
From: Den <motometalor@gmail.com>
Date: Sat, 26 Jan 2019 12:41:17 +0300
Subject: [PATCH 3/3] xt907_jbbl: switch from msm8960-common -> to
 msm8960_jbbl-common

According to rebranding xt907 -> to xt907_jbbl
---
 xt907_jbbl/xt907_jbbl-vendor.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xt907_jbbl/xt907_jbbl-vendor.mk b/xt907_jbbl/xt907_jbbl-vendor.mk
index e75c0002..f73b7aab 100644
--- a/xt907_jbbl/xt907_jbbl-vendor.mk
+++ b/xt907_jbbl/xt907_jbbl-vendor.mk
@@ -18,6 +18,6 @@
 DEVICE_PACKAGE_OVERLAYS += vendor/motorola/xt907_jbbl/overlay
 
 $(call inherit-product, vendor/motorola/xt907_jbbl/xt907_jbbl-vendor-blobs.mk)
-$(call inherit-product, vendor/motorola/msm8960-common/msm8960-common-vendor.mk)
+$(call inherit-product, vendor/motorola/msm8960_jbbl-common/msm8960_jbbl-common-vendor.mk)
 $(call inherit-product, vendor/motorola/qcom-common/qcom-common-vendor.mk)
 
-- 
2.17.1

